src/main/kotlin/org/phoenixframework/PhxPush.kt

```diff
@@ -185,7 +185,7 @@ class PhxPush(
         mutPayload["status"] = status
 
         refEvent?.let {
-            val message = PhxMessage(it, "", "", mutPayload)
+            val message = PhxMessage(it, "", it, mutPayload)
```

Comment:
Let's make a change to clean this up a bit and then it should be good to go

```Kotlin
refEvent?.let { safeRefEvent ->
    val message = PhxMessage("", "", safeRefEvent, mutPayload)
    this.channel.trigger(message)
}
```

Reply:
I fixed it up but opted to use named parameters instead of passing in the empty string for two of them